### DISCUSSION TITLE
How can I recognize an AID easily and reliably - QR code?

### DISCUSSION THREAD

**Discussion by edeykholt** [trustoverip/kswg-keri-specification#185] [General]
What is the specification for an AID represented in a text QR code?

**Comment by nkongsuwan** [comment]
Is this a question about (1) what data should be encoded into a QR code or (2) how to visually represent a QR code so that its receiver knows which AID the QR code represents?

For (1), the simplest way is probably to convert an OOBI URL of the AID into a QR code. Then, the other party could scan and resolve the OOBI URL. 

Should other metadata be included? Perhaps, the QR code could include auxiliary information such as the personal details of the QR code presenter that helps the receiver to know to whom this QR code belongs. Could the auxiliary information be exploited by social engineering?
